tests/lib/hw_unique_key_tfm/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+if (NOT DEFINED HUK_BOARD_SNR)
+	message(WARNING "If it is to be run, this test requires setting the board serial number by -DHUK_BOARD_SNR=<SEGGER num> to perform a custom flashing algorithm on the KMU.")
```

Comment:
could you split this line ?
you can do:
```
message(WARNING "If it is to be run, this test requires setting"
  "the board serial number by -DHUK_BOARD_SNR=<SEGGER num>"
  "to perform a custom flashing algorithm on the KMU."
)
```

Feel free to split where you like.